tests/meson.build

```diff
@@ -353,9 +353,10 @@ ctags_tests = files([
 	'ctags/whitespaces.php.tags'
 ])
 
+fs = import('fs')
 runner = find_program('ctags/runner.sh')
 foreach t : ctags_tests
-	test('@0@'.format(t), runner,
+	test(join_paths(fs.parent(t), fs.name(t)), runner,
```

Comment:
The required meson version is 0.53 and this feature is not available until 0.59

Looks like they've added full_path but that feature requires meson 1.4.1

https://stackoverflow.com/a/77647927/6838037

https://mesonbuild.com/Reference-manual_returned_file.html

https://github.com/mesonbuild/meson/issues/12259

Anyway, a pretty trivial issue. It works as-is. I assume you don't wanna set the required meson version to 0.59 yet. Feel free to close this PR unless you wanna discuss it further.

Reply:
One other potentially noteworthy feature of 0.59 is that when using 'ninja uninstall`, the gmo/mo files get uninstalled as they should.

Reply:
@eli-schwartz Isn't there a proper way to convert a file object to a string in <=0.53? (I suppose that's why the warning isn't a hard error, and the fs and full_path() features have been introduced).

Reply:
You could start off with strings. :)